modules/nw-allocate-load-balancers-to-subnets-procedure.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// *networking/allocating-load-balancers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-allocating-load-balancers-to-subnets-procedure_{context}"]
+= Specifying AWS Subnets for OpenShift Ingress Load Balancers at Installation
+
+Perform the following steps to allocate API and Ingress load balancers to specific subnets.
+
+.Prerequisites
+
+Before you begin, ensure you have:
+
+* An existing AWS virtual private cloud (VPC).
+
+* Pre-configured AWS subnets intended for use by the OpenShift cluster, with the following considerations:
+
+** A list of their subnet IDs (for example, `subnet-0123456789abcdef0`). These IDs will be used in the `install-config.yaml` file.
+
+** For the default public Ingress Controller, any subnet assigned the `IngressControllerLB` role in your `install-config.yaml` file must be a public subnet. For example, it must have a route table entry in AWS that directs outbound traffic to an internet gateway (IGW).
```

Comment:
nit: I think we can use a similar wording like [here](https://github.com/openshift/openshift-docs/pull/91360/files#diff-1914b30a1dfc262059fcedb73473601087e9778910ee8199a2e7feb5730ef441R85)

> In public/external cluster, the subnet with `IngressControllerLB` role in your `install-config.yaml` file must be public. For example, it must have a route table entry in AWS that directs outbound traffic to an internet gateway (IGW).

Or we can just move this paragraph to the [same place](https://github.com/openshift/openshift-docs/pull/91360/files#diff-1914b30a1dfc262059fcedb73473601087e9778910ee8199a2e7feb5730ef441R85).